src/css/components/po-tag/po-tag.css

```diff
@@ -190,6 +194,15 @@
   color: var(--text-color-warning);
 }
 
+.po-clickable.po-tag-wrapper,
+.po-click.po-tag-wrapper {
+  min-width: 44px;
+  min-height: 44px;
+  display: flex;
+  align-items: center;
+  outline: none;
+}
```

Comment:
`justify-content: center;`
para quando o tamanho da tag for menor que 44px, ele ficar centralizado